src/NzbDrone.Core.Test/ParserTests/LanguageParserFixture.cs

```diff
@@ -198,6 +198,25 @@ public void should_parse_language_polish_subbed(string postTitle)
             result.Languages.Should().BeEquivalentTo(Language.Unknown);
         }
 
+        [TestCase("Movie.Title.1994.1080p.UA.XviD")]
+        [TestCase("Movie.Title.1994.1080p.UKR.XviD")]
+        [TestCase("Movie.Title.1994.1080p.2xUKR.XviD")]
+        public void should_parse_language_ukrainian(string postTitle)
+        {
+            var result = Parser.Parser.ParseMovieTitle(postTitle, true);
+
+            result.Languages.Should().BeEquivalentTo(Language.Ukrainian);
+        }
+
+        [TestCase("Movie.Title.1994.H.265.[UKR_ENG]")]
```

Comment:
same as above; nothing on predb

Reply:
This notation although rare, sometimes appears on private trackers.